### PR TITLE
ci: added pre-release workflow

### PR DESCRIPTION
**Pull request by iadmytro**
Added CI pre-release workflow

### CODE REVIEW

File: .github/workflows/pre-release.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        run: |
+          yarn install --frozen-lockfile --non-interactive --logevel=error
+          yarn build
+
+      - name: 🔬 Test
+        working-directory: packages/wallet
+        env:
+          BLOCKFROST_API_KEY: ${{ secrets.BLOCKFROST_API_KEY }}
+          MNEMONIC_WORDS_ONE: ${{ secrets.MNEMONIC_WORDS_ONE }}
+          MNEMONIC_WORDS_TWO: ${{ secrets.MNEMONIC_WORDS_TWO }}
```
Comment (james-iohk) [inline .github/workflows/pre-release.yaml:35]: We only want `MNEMONIC_WORDS`

**Review by james-iohk — CHANGES_REQUESTED** [review]

File: .github/workflows/pre-release.yaml (hunk 1 above)
Comment (mkazlauskas) [reply .github/workflows/pre-release.yaml:35]: Also seems like we're missing some variable(s) in env because the check on this PR failed

**Review by Newman1515 — COMMENTED** [review]
Submit the approve
And
Submit the review

**Comment by rhyslbw** [comment]
We now have a _release_ workflow, so this one is no longer relevant.

**Pull request closed without merging**